website/project/tasks.py

```diff
@@ -10,6 +16,29 @@ def on_node_updated(node_id, saved_fields, request_headers=None):
         if node.check_spam(saved_fields, request_headers):
             node.save()
 
+            if settings.SHARE_URL and settings.SHARE_API_TOKEN:
```

Comment:
Might want to pull this out form under the spam check and rather look at saved_fields for `spam_status`, so when a node update comes through form an admin `confirm_spam` or `confirm_ham` we can publish/delete as needed to SHARE.